notebooks/data-sources/TestGrid/metrics/metric_template.ipynb

```diff
@@ -315,9 +316,16 @@
     "                            # time duration values for this test and setting it to 'None'\n",
     "                            # for all the other tests in each grid\n",
     "                            if tests[i] == \"Overall\":\n",
-    "                                test_duration = graphs[i][0][\"values\"][0][j]\n",
-    "                            else:\n",
-    "                                test_duration = None\n",
+    "                                try:\n",
+    "                                    test_duration = graphs[i][0][\"values\"][0][j]\n",
+    "                                except IndexError:\n",
+    "                                    print(\n",
+    "                                        \"Exception occurred in test_duration calculation\",\n",
```

Comment:
Can you provide a more descriptive statement? like "No values present to calculate test duration" or something? The current comment sounds a little vague, and unclear if the user should be concerned  by exception.

Reply:
Yes, will do 👍

Reply:
made the change!